backend/src/main/java/com/ody/notification/service/NotificationService.java

```diff
@@ -41,21 +46,23 @@ public void saveAndSendNotifications(
             DeviceToken deviceToken,
             RouteTime routeTime
     ) {
-        saveAndSendEntryNotification(meeting, mate);
-        fcmSubscriber.subscribeTopic(new FcmTopic(meeting), deviceToken);
-        saveAndSendDepartureReminderNotification(meeting, mate, routeTime);
+        saveAndSendEntryNotification(mate);
+        FcmTopic fcmTopic = new FcmTopic(meeting);
+        fcmSubscriber.subscribeTopic(fcmTopic, deviceToken);
+        saveAndSendDepartureReminderNotification(meeting, mate, routeTime, fcmTopic);
     }
 
-    private void saveAndSendEntryNotification(Meeting meeting, Mate mate) {
+    private void saveAndSendEntryNotification(Mate mate) {
         Notification notification = Notification.createEntry(mate);
-        saveAndSendNotification(meeting, notification);
+        saveAndSendNotification(notification);
     }
 
-    private void saveAndSendDepartureReminderNotification(Meeting meeting, Mate mate, RouteTime routeTime) {
+    private void saveAndSendDepartureReminderNotification(Meeting meeting, Mate mate, RouteTime routeTime,
+                                                          FcmTopic fcmTopic) {
```

Comment:
[제안]

개행이 됐네요! 컨벤션 맞춰주세요~

```suggestion
    private void saveAndSendDepartureReminderNotification(
        Meeting meeting,
        Mate mate,
        RouteTime routeTime,
        FcmTopic fcmTopic
    ) {
```

Reply:
인텔리제이가 자동으로 감지하고 정렬해줬으면 ㅠㅠ 😭
반영완료 !